pkg/cloudprovider/fake/fake.go

```diff
@@ -129,14 +131,19 @@ func (f *FakeCloud) NodeAddresses(instance string) ([]api.NodeAddress, error) {
 	return f.Addresses, f.Err
 }
 
-// ExternalID is a test-spy implementation of Instances.ExternalID.
+// GetInstanceID is a test-spy implementation of Instances.GetInstanceID.
 // It adds an entry "external-id" into the internal method call record.
```

Comment:
you should change the call name from "external-id" to "instance-id" to match the new method name